aws/eks/main.tf

```diff
@@ -61,20 +61,15 @@ module "eks" {
 
   cluster_addons = {
     coredns = {
-      version  = "v1.11.3-eksbuild.1"
```

Comment:
if we don't pin the versions it will drift each time a new version is available.

Reply:
or not?